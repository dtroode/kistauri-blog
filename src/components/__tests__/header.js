import React from "react";
import renderer from "react-test-renderer";

import Header from "../header";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Header pageClass="portfolio" title="Давид Кистаури >_<" link="" />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
