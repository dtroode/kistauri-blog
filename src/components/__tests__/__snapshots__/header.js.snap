// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="head--portfolio"
>
  <nav
    className="head__nav"
  >
    <a
      className="head__nav__a a--primary a--active"
      href="/"
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    >
      &gt;_&lt;
    </a>
    <a
      className="head__nav__a a--primary"
      href="/blog"
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    >
      Блог
    </a>
  </nav>
  <h1
    className="head__header"
  >
    Давид Кистаури &gt;_&lt;
  </h1>
  <hr />
</header>
`;
